utils/DeployUtils.ps1

```diff
@@ -181,31 +217,25 @@ function ConfigureScubaGearModule {
         [string]
         $PrereleaseTag = ""
     )
-    Write-Host "Configuring ScubaGear module..."
-    #TODO: Add any module configuration needed (e.g., adjust Module Version)
+    Write-Warning ">>> Configuring ScubaGear module..."
 
+    #TODO: Add any module configuration needed (e.g., adjust Module Version)
```

Comment:
Is this TODO item tracked anywhere else? Just flagging it to make sure it doesn't get lost.